.github/workflows/verify_docs-quality.yml

```diff
@@ -30,6 +30,6 @@ jobs:
         with:
           # This also contains --config=.github/vale/config.ini ... :/
           files: '${{ steps.generate.outputs.args }}'
-          version: 2.30.0 # TODO: migrate to v3
+          version: 3.0.4
```

Comment:
I guess this is a point of interest. Whether we should pin the version to not have to worry about major shifts in the future, but to also lose out on updates to vale. Which is worse? Maybe we can assume that it'll take a long time before the next major that has a breakage that actually affects our current setup?